### PR TITLE
Task Track - Edit

### PR DESCRIPTION
**Pull request by leonidbogin**
image

### CODE REVIEW

**Review by SashaDrapun — APPROVED** [review]
Good job. I didn't find any errors.

File: DIMS-Core/DIMS-Core/Controllers/TaskTrackController.cs [hunk 1]
```diff
@@ -74,6 +74,53 @@ public async Task<IActionResult> Create([FromForm] TaskTrackViewModel model)
...
+        [HttpGet("edit/{id}")]
+        public async Task<IActionResult> Edit(int id, string back = null)
+        {
+            if (id <= 0)
+            {
+                return BadRequest();
+            }
+
+            // To Do - Get the id of the current user
+            int userId = 3;
```
Comment (maximsan) [inline DIMS-Core/DIMS-Core/Controllers/TaskTrackController.cs:86]: why do we need it ?

File: DIMS-Core/DIMS-Core/Controllers/TaskTrackController.cs [hunk 2]
```diff
@@ -74,6 +74,53 @@ public async Task<IActionResult> Create([FromForm] TaskTrackViewModel model)
...
+            ViewBag.BackController = back;
+            return View(model);
+        }
+
+        [HttpPost("edit")]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> Edit([FromForm] TaskTrackViewModel model)
+        {
+            // To Do - Get the id of the current user
+            int userId = 3;
```
Comment (maximsan) [inline DIMS-Core/DIMS-Core/Controllers/TaskTrackController.cs:102]: who will add the logic ?

**Review by maximsan — COMMENTED** [review]
Looks good for me except hardcoded id. What does it mean ?

**Review by PushMeNow — APPROVED** [review]

File: DIMS-Core/DIMS-Core/Controllers/TaskTrackController.cs (hunk 1 above)
Comment (leonidbogin) [reply DIMS-Core/DIMS-Core/Controllers/TaskTrackController.cs:86]: I'm waiting for an approval pull requests 23 [https://github.com/Dev-incubator/DIMS-Core/pull/23].
The required functionality is in that pr.

File: DIMS-Core/DIMS-Core/Controllers/TaskTrackController.cs (hunk 2 above)
Comment (leonidbogin) [reply DIMS-Core/DIMS-Core/Controllers/TaskTrackController.cs:102]: I'm waiting for an approval pull requests 23 [https://github.com/Dev-incubator/DIMS-Core/pull/23].
The required functionality is in that pr.

**Pull request merged**